app/assets/stylesheets/components/_inputs.scss

```diff
@@ -0,0 +1,27 @@
+input[type=text] {
+    width: 300px;
```

Comment:
@RamZallan In these base styles, try not to set widths whenever possible.  It's better to leave them flexible in the base style and then we can add a layer on it on a per-use basis, if necessary.